exercises/input-output/src/main/java/com/cbfacademy/ReadInvertedFile.java

```diff
@@ -0,0 +1,58 @@
+package com.cbfacademy;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.FileAlreadyExistsException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+public class ReadInvertedFile {
+    // // create a programme to invert the exercise.txt file
+    public void TextInverter (String fileName, String inverterdPath) throws IOException{
```

Comment:
if you're catching the IOEXception within our method, so you need this `throws` here?